### PR TITLE
Feature/ Implemented the ability to navigate between pages

### PR DESCRIPTION
**Pull request by svtslv17**
max@gmail.com
user@example.com
admin@gmail.com
li@gmail.com

Password

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
+import { Route, Redirect, useLocation } from 'react-router-dom';
 import { connect } from 'react-redux';
 import { ThemeProvider } from '@mui/material/styles';
 import { theme } from './utils';
-import { Home, Login, Internships } from './pages';
+import { Home, Login } from './pages';
 
 function App(props) {
   const { isAuthorized } = props;
+  const currentPath = useLocation().pathname;
```
Comment (nsukhotskaya) [inline src/App.jsx:11]:
```suggestion
  const location = useLocation();
  const currentPath = location.pathname;
```

File: src/components/header/headerMenu/HeaderMenu.jsx [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
 import { getFieldLabel } from '../../../utils';
 
 function HeaderMenu() {
   return (
     <ButtonGroup variant="text" aria-label="text button group" color="primary">
-      <Button>{getFieldLabel('header.menu.button.internships')}</Button>
-      <Button>{getFieldLabel('header.menu.button.candidates')}</Button>
-      <Button>{getFieldLabel('header.menu.button.letters')}</Button>
+      <Button>
+        <Link to="/internships">
```
Comment (nsukhotskaya) [inline src/components/header/headerMenu/HeaderMenu.jsx:10]: Please read about buttons and links. https://css-tricks.com/a-complete-guide-to-links-and-buttons/

File: src/pages/home/Home.jsx [hunk 3]
```diff
@@ -1,27 +1,41 @@
...
+                <ProfileCard />
+              </Route>
+              <Route exact path="/internships">
+                <Internships />
+              </Route>
+              <Route exact path="/candidates">
+                <Candidates />
+              </Route>
+            </Switch>
+          </Box>
```
Comment (nsukhotskaya) [inline src/pages/home/Home.jsx:33]: Why is the footer in the main block?

File: src/App.jsx (hunk 1 above)
Comment (nsukhotskaya) [inline src/App.jsx:11]:
```suggestion
  const location = useLocation();
```

Comment (itsnotpolza) [inline src/App.jsx:11]:
```suggestion
  const { pathname } = useLocation();
```
This way it's more readable

File: src/components/header/headerMenu/HeaderMenu.jsx (hunk 2 above)
Comment (nsukhotskaya) [reply src/components/header/headerMenu/HeaderMenu.jsx:10]: Also pay attention https://mui.com/api/button/
image

**Review by nsukhotskaya — APPROVED** [review]

**Review by OstroumovAlex — APPROVED** [review]

**Pull request merged**